README.md

```diff
@@ -7,22 +7,301 @@ Serlio requires so-called rule packages (RPK) as input, which are authored in Ci
 Serlio is well suited for managing the procedural generation of architectural 3D content in digital sets. However, Serlio is restricted to the procedural generation of single buildings / objects. Serlio does not include the city layouting and street network editing tools of CityEngine i.e. the rich CityEngine toolset to design a city from scratch (or based on geographic data) is still needed.
 
 
-## Documentation
+## Quick Start
+1. Use CityEngine to [export a Rule Package](https://doc.arcgis.com/en/cityengine/latest/help/help-rule-package.htm) (RPK).
+1. Acquire the Serlio binaries, either: 
+    * ... download the [desired release](https://github.com/Esri/serlio/releases)
+    (on linux you still need to manually [install the local build](#linux-2))
+    * ... or [build](#build-instructions) them yourself, which will result in an `install` directory
```

Comment:
```suggestion
1. Install Serlio by downloading the latest [installer](https://github.com/Esri/serlio/releases/latest).
```

Reply:
Changed to 
1. Install Serlio by downloading the latest [installer](https://github.com/Esri/serlio/releases/latest). Please see [below](#linux-2) for installation on linux.